example/android/app/build.gradle

```diff
@@ -109,6 +109,16 @@ dependencies {
     implementation("com.facebook.react:react-android")
     implementation("com.facebook.react:flipper-integration")
 
+    /**
+     * androidx.activity:activity >= 1.7.0 is required for the backported photo picker to work on Android < 30
+     * https://developer.android.com/jetpack/androidx/releases/activity#1.7.0.
+     *
+     * This dependency can be removed from here once the react-native library updates androidx.appcompat:appcompat to >= 1.7.0
+     * https://github.com/facebook/react-native/blob/3dfe22bd27429a43b4648c597b71f7965f31ca65/packages/react-native/gradle/libs.versions.toml#L11
+     * because androidx.appcompat:appcompat:1.7.0 depends on androidx.activity:activity:1.7.0 https://developer.android.com/jetpack/androidx/releases/appcompat#1.7.0
+     */
+    implementation("androidx.activity:activity:1.7.+")
```

Comment:
Hi @hubatruck 
I actually just discovered that the multi select is not working on older Android devices. It looks like there has been several fixes to the photo picker since v1.7, and specifically v1.8.2 fixed the multi select on older Android devices:
https://developer.android.com/jetpack/androidx/releases/activity#1.8.2

So I think we should change
```
implementation("androidx.activity:activity:1.7.+")
```
to
```
implementation("androidx.activity:activity:1.9.+")
```
And then update the comment section here as well as the README. (there is not yet a version of `androidx.appcompat:appcompat` that depends on `androidx.activity:activity:1.9.+`)

Reply:
I have updated the docs, and upon testing, seems like it works with the new version too, and fixes the multiple selection bug